### ISSUE TITLE
 Who have seen a message and support it

### ISSUE THREAD

**Issue by szwss** [antoine92190/vue-advanced-chat#418]
There are multiple members in a chat room, and I want to know a message I sent, which members have seen it,
Currently I use this package, but I found that seen is a boolean value, and I can only know if it has been viewed, not who has seen it 
Can it be modified to support this feature

**Comment by antoine92190** [comment]
Yes it's a nice feature, I'll consider it as a future development

**Comment by palmtown** [comment]
Hello @antoine92190 

I hope all is well.  I just wanted to check in to see if you have planned this feature in the near future.  I too, would like to see this available.

Thanks!
